lib/spatial_features/importers/file.rb

```diff
@@ -3,22 +3,40 @@
 module SpatialFeatures
   module Importers
     class File < SimpleDelegator
-      def initialize(data, *args)
+      INVALID_ARCHIVE = "Archive did not contain a .kml or .shp file. Supported formats are KMZ, KML, and zipped ArcGIS shapefiles.".freeze
+
+      def initialize(data, *args, **options)
+        # the current_file param is passed by `::create` after it has opened a zip
```

Comment:
I wonder if there's a universe in which we can receive a zip of anything and divvy the files up as required. I see the issue though. That's a thing the happens high up, but it needs each of the Importer classes to be able to identify which files they need. Just food for thought I guess.

Reply:
We're not far from that universe.  I added a spec proving that a zip with multiple KMLs parsed just fine as is.  

Do you want to take it further, or let it settle like this and see if there's a use case down the road?